src/borg/testsuite/archiver/check_cmd.py

```diff
@@ -92,6 +92,11 @@ def test_date_matching(self):
         self.assert_in("archive2", output)
         self.assert_not_in("archive3", output)
 
+        # check for output when timespan older than earliest archive is given. Issue #1711
+        output = self.cmd(
+            f"--repo={self.repository_location}", "check", "-v", "--archives-only", "--older=9999m", exit_code=0
+        )
+
```

Comment:
ok, that makes sure nothing crashed.

you could also test that it gives the expected result (all archives "not in" the output).

Reply:
Yes, added.